docs/src/main/asciidoc/datastore.adoc

```diff
@@ -529,6 +527,26 @@ The GQL key-literal representation for the parent `ShoppingOrder` would be:
 Key(orders, '12345')
 ----
 
+The Cloud Datastore entities exist separately in their own kinds:
+
+
+----
+{
```

Comment:
Can you indicate that this is the order and that the two below are purchased_item?

Reply:
Done, PTAL